lib/loggers/index.js

```diff
@@ -250,104 +224,30 @@ container.init = async () => {
       // set up message handler for master process
       worker.on('message', function(msg) {
         if(typeof msg === 'object' && msg.type === 'bedrock.logger') {
-          container.get(msg.category).log(msg.level, msg.msg, msg.meta);
+          container.get(msg.category).log(msg.info);
         }
       });
     };
 
     return;
   }
 
-  // workers:
-
-  // define transport that transmits log message to master logger
-  const WorkerTransport = function(options) {
-    winston.Transport.call(this, options);
-    this.category = options.category;
-  };
-  util.inherits(WorkerTransport, winston.Transport);
-  WorkerTransport.prototype.name = 'worker';
-  WorkerTransport.prototype.log = function(level, msg, meta, callback) {
-    if(this.silent) {
-      return callback(null, true);
-    }
-
-    // FIXME: rework this error handling and callers
-    // Add support for BedrockError and its toObject() feature.
-
-    // pull out any meta values that are pre-formatted
-    meta = meta || {};
-    let preformatted = null;
-    const metaIsObject = brUtil.isObject(meta);
-    let module = null;
-    if(metaIsObject) {
-      if('preformatted' in meta) {
-        preformatted = meta.preformatted;
-        delete meta.preformatted;
-      }
-      if('module' in meta) {
-        module = meta.module;
-        delete meta.module;
-      }
-    }
-    // stringify and include the worker PID in the meta information
-    let json;
-    try {
-      json = JSON.stringify(meta, null, 2);
-    } catch(e) {
-      json = JSON.stringify(cycle.decycle(meta), null, 2);
-    }
-    let error;
-    if(meta instanceof Error) {
-      error = ('stack' in meta) ? meta.stack : meta;
-      meta = {error, workerPid: process.pid, workerId};
-    } else if(metaIsObject && 'error' in meta) {
-      error = ('stack' in meta.error) ? meta.error.stack : meta.error;
-      meta = {error, workerPid: process.pid, workerId};
-    } else {
-      meta = {workerPid: process.pid, workerId};
-    }
-
-    // only add details if they exist
-    if(json !== '{}') {
-      meta.details = json;
-    }
```

Comment:
I am a little concerned we're losing some kind of functionality or corner case handling here. Maybe it's fine. Your thoughts, @mattcollier?

Reply:
Maybe this is all handled by the formatters/fast JSON stringify stuff I saw elsewhere.

Reply:
This is some of the legacy stuff I want to flush out.  If it's not being handled by formatters properly now, we can iteratively fix things up.